sci-biology/spm/spm-8.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+DESCRIPTION="Analysis of brain imaging data sequences for Octave or Matlab"
+HOMEPAGE="http://www.fil.ion.ucl.ac.uk/spm/"
+URI_BASE="http://www.fil.ion.ucl.ac.uk/spm/download/restricted/idyll/"
+SRC_URI="${URI_BASE}${PN}${PV}.zip"
+
+LICENSE="GPL-2+"
+SLOT="8"
```

Comment:
I think what @junghans suggests is using `SLOT="0"` for spm8 and `SLOT="1"` for spm12.